curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-string-and-array-methods-by-building-a-music-player/655235c2e607297f00316650.md

```diff
@@ -0,0 +1,650 @@
+---
+id: 655235c2e607297f00316650
+title: Step 26
+challengeType: 0
+dashedName: step-26
+---
+
+# --description--
+
+Next, use the `classList` property and the `add()` method to add the `playing` class to the `playButton` element.
+
+To finally play the song, use the `play()` method on the `audio` variable.
+
+# --hints--
+
+Test 1
+
+```js
+
+```
```

Comment:
The tests:
You should use the `classList` property and the `add` method to add the class `playing` to `playButton`.

```js
assert.match(code, /playButton\.classList\.add\(("|')playing\1\);?/)
```

You should use the `play()` method on the `audio` variable.

```js
assert.match(code, /audio\.play\(\);?/)
```